apps/prairielearn/elements/pl-drawing/pl-drawing.mustache

```diff
@@ -1,5 +1,5 @@
 {{#render_button}}
-<button type="button" name="{{button_class}}" class="btn btn-primary pl-drawing-button" opts="{{{ options }}}"><img /></button>
+<button type="button" name="{{button_class}}" class="btn btn-primary pl-drawing-button" opts="{{{ options }}}" title="{{button_label}}"><img src=""></button>
```

Comment:
The image source is set via JS, so this just makes the linter happy. I double checked that it still set the source correctly if it was defined.